common/pylintrc

```diff
@@ -79,6 +79,7 @@ disable=
   too-many-branches,
   too-many-arguments,
   too-many-locals,
+  too-many-nested-blocks,
```

Comment:
sounds like a good test to keep, maybe you can just white list the offending lines

Reply:
Fixed, good point.